testsuite/features/support/api_test.rb

```diff
@@ -83,12 +102,22 @@ def date_now
 
 # Derived class for an HTTP test
 class ApiTestHttp < ApiTest
+  ##
+  # It creates a new instance of the HttpClient class.
+  #
+  # Args:
+  #   host: The hostname of the server.
   def initialize(host)
     super
     @connection = HttpClient.new(host)
   end
 
-  # during HTTP tests, dates are strings
+  ##
+  # XML-RPC uses Date class for dates, while HTTP RPC uses simple strings for dates.
+  # This function provides a string containing a date that can be swallowed by HTTP RPC.
```

Comment:
@Bischoff this function seems to just return a boolean and not a string. Same for the other `date?` function some methods above - it's not clear to us how these work exactly

Reply:
ok, there was some confusion indeed, probably my bad.

there are 2 functions:
```
  def date?(attribute)
```
as HTTP API uses strings for dates, it's not obvious if a call to that API returned a date or something else. that first function returns a boolean which is true if it's indeed a date.

```
  def date_now()
```
as HTTP API uses strings for dates, we need a function to construct a string out of current date, to be passed as a parameter to an API call


hope it is clearer now